src/App.js

```diff
@@ -202,14 +202,18 @@ function App() {
     },
     [push]
   );
+
```

Comment:
we also need to update it here https://github.com/XLabs/portal-bridge-ui/blob/main/public/sui/index.html#L44

Reply:
I think it's ready